archivist/src/main/scala/uk/ac/wellcome/platform/archive/archivist/bag/ArchiveJobCreator.scala

```diff
@@ -85,7 +86,7 @@ object ArchiveJobCreator extends Logging {
                                ingestBagRequest: IngestBagRequest)
     : Either[ArchiveError[IngestBagRequest], ExternalIdentifier] = {
     ZipFileReader
-      .maybeInputStream(ZipLocation(zipFile, BagItemPath(bagInfoPath)))
+      .maybeInputStream(ZipEntryPointer(zipFile, bagInfoPath))
```

Comment:
minor: would it be better to pass a ZipEntryPointer directly into this method?

Or if not, clearer to rename this parameter `zipInfoPath`?